features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,635 @@
+      | aanduidingBijzonderNederlanderschap | <bijz. NL-schap>             |
+      | datumIngangGeldigheid               | <datumIngangGeldigheid>      |
+
+      Voorbeelden:
+      | burgerservicenummer | nationaliteit | nationaliteit omschrijving | bijz. NL-schap | redenOpname | redenOpname omschrijving                        | datumIngangGeldigheid |
+      | 000000450           | 0001          | Nederlandse                |                | 001         | Wet op het Nederlanderschap 1892, art.1, lid 1a | 19750707              |
+      | 000000462           | 0263          | Surinaamse                 |                | 301         | Vaststelling bezit vreemde nationaliteit        | 00000000              |
+      | 000000474           | 0000          | Onbekend                   |                | 311         | Vaststelling onbekende nationaliteit            | 00000000              |
+      | 000000486           |               |                            | B              | 310         | Vaststelling bijzonder Nederlanderschap         | 19570115              |
+      | 000000498           |               |                            | V              | 310         | Vaststelling bijzonder Nederlanderschap         | 19750615              |
+
+  Rule: wanneer een veld van een datum wordt gevraagd, wordt de datum geleverd
+
+    Abstract Scenario: met fields is gevraagd om <veld>.<datumveld>
+      Gegeven de persoon met burgerservicenummer '000000504' heeft de volgende 'inschrijving' gegevens
+      | naam                | waarde    |
+      | <GBA veld>          | 20131102  |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000504                       |
+      | fields              | <veld>.<datumveld>              |
+      Dan heeft de response een persoon met alleen de volgende gegevens
+      | naam   | waarde   |
+      | <veld> | 20131102 |
+
+      Voorbeelden:
+      | datumveld   | GBA veld                                  | veld                        |
+      | type        | datum eerste inschrijving GBA (68.10)     | datumEersteInschrijvingGBA  |
+      | langFormaat | datum eerste inschrijving GBA (68.10)     | datumEersteInschrijvingGBA  |
+      | datum       | datum eerste inschrijving GBA (68.10)     | datumEersteInschrijvingGBA  |
+      | jaar        | datum eerste inschrijving GBA (68.10)     | datumEersteInschrijvingGBA  |
+      | maand       | datum eerste inschrijving GBA (68.10)     | datumEersteInschrijvingGBA  |
+      | onbekend    | datum eerste inschrijving GBA (68.10)     | datumEersteInschrijvingGBA  |
+      | type        | datum inschrijving in de gemeente (09.20) | datumInschrijvingInGemeente |
+      | langFormaat | datum inschrijving in de gemeente (09.20) | datumInschrijvingInGemeente |
```

Comment:
datumInschrijvingGemeente is een "verblijfplaats gegeven en geen ïnschrijving"gegeven. Deze inserts gaan verkeerd. (op regel 555 t/m 560)

Reply:
Is aangepast, de groep is variabel gemaakt.

> Mijn gevoel zegt mij dat deze feature moet beschrijven/valideren dat de gba de mapping implementeert die in de fields-mapping-gba.csv is gedefinieerd, maar ik kom bijv. geen scenario tegen dat beschrijft dat voor aanhef en aanschrijfwijze de aangaanHuwelijkPartnerschap.datum en ontbindingHuwelijkPartnerschap.datum moet worden geretourneerd

Moet worden toegevoegd. Ik vermoed dat er nog wel enkele scenario's meer zullen moeten worden toegevoegd.
Lijkt me goed om daar een apart issue voor aan te maken zodat het mergen van deze versie van informatievragen-fields-gba.feature daar niet op hoeft te wachten.

Reply:
Is aangepast, de categorie is variabel gemaakt.